packages/orchestrator/internal/sandbox/fc/process.go

```
@@ -452,6 +475,13 @@ func (p *Process) Stop() error {
 	return nil
 }
 
+func getProcessState(pid int) (string, error) {
+	cmd := exec.Command("ps", "-o", "stat=", "-p", fmt.Sprint(pid))
+	out, _ := cmd.Output()
```

the error is ignored by design?